docker/elasticsearch-setup/ml-model-index-config.json

```diff
@@ -0,0 +1,212 @@
+            "filter": [
+              "lowercase"
+            ],
+            "type": "custom",
+            "tokenizer": "comma_tokenizer"
+          },
+          "custom_browse": {
+            "filter": [
+              "lowercase"
+            ],
+            "type": "custom",
+            "tokenizer": "path_hierarchy_tokenizer"
+          },
+          "custom_ngram": {
+            "filter": [
+              "lowercase"
+            ],
+            "type": "custom",
+            "tokenizer": "custom_ngram"
+          },
+          "custom_keyword": {
+            "filter": [
+              "lowercase"
+            ],
+            "type": "custom",
+            "tokenizer": "keyword"
+          },
+          "comma_pattern_ngram": {
+            "filter": [
+              "lowercase",
+              "autocomplete_filter"
+            ],
+            "type": "custom",
+            "tokenizer": "comma_tokenizer"
+          },
+          "delimit": {
+            "filter": [
+              "lowercase",
+              "custom_delimiter"
+            ],
+            "tokenizer": "whitespace"
+          },
+          "ml_model_pattern_ngram": {
+            "filter": [
+              "lowercase",
+              "autocomplete_filter"
+            ],
+            "type": "custom",
+            "tokenizer": "ml_model_pattern"
+          },
+          "custom_browse_slash": {
+            "filter": [
+              "lowercase"
+            ],
+            "type": "custom",
+            "tokenizer": "path_hierarchy"
+          }
+        },
+        "tokenizer": {
+          "path_hierarchy_tokenizer": {
+            "type": "path_hierarchy",
+            "replacement": "/",
+            "delimiter": "."
+          },
+          "custom_ngram": {
+            "type": "ngram",
+            "min_gram": "3",
+            "max_gram": "50"
+          },
+          "slash_tokenizer": {
+            "pattern": "[/]",
+            "type": "pattern"
+          },
+          "comma_tokenizer": {
+            "pattern": ",",
+            "type": "pattern"
+          },
+          "ml_model_pattern": {
+            "pattern": "[./]",
+            "type": "pattern"
+          }
+        }
+      }
+    }
+  },
+  "mappings": {
+    "doc": {
+      "properties": {
+        "browsePaths": {
+          "type": "text",
+          "fields": {
+            "length": {
+              "type": "token_count",
+              "analyzer": "slash_pattern"
+            }
+          },
+          "analyzer": "custom_browse_slash",
+          "fielddata": true
+        },
+        "origin": {
+          "type": "keyword",
+          "fields": {
+            "ngram": {
+              "type": "text",
+              "analyzer": "custom_ngram"
+            }
+          },
+          "normalizer": "my_normalizer"
+        },
+        "hasOwners": {
+          "type": "boolean"
+        },
+        "name": {
+          "type": "keyword"
+        },
+        "num_inputs": {
```

Comment:
there seems to a disconnect b/w the mapping of mlModel and the fields defined in `MLModelDocument`. For instance field `num_inputs`, `num_outputs`, `inputs`, `outputs`, etc fields are not defined in the search document model. Similarly there are fields in the search document model `MLModelDocument` such as `trainingDatasets` which aren't provided in this mapping file. Could you please help clarify?

Reply:
I forgot to change the fields after copying from another model, will fix that.